mail_activity_team/models/mail_activity_team.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class MailActivityTeam(models.Model):
+    _name = "mail.activity.team"
```

Comment:
can't we recycle some other already existing model here? Seems a bit of a hassle to have to admin those separately

Reply:
@hbrunn hr department would be a good match, but would require depending on hr, and also sometimes a department can have several teams